github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -428,7 +428,7 @@ presubmits:
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
     always_run: true
     optional: true
```

Comment:
is it just me or the optional: true line that was just changed isnt marked ?
guess its github problems, and it means we need to make sure what we see is what we get

Reply:
You are not seeing things. The previous patch was editing the IPv6 job. Not sure how I messed that up.